StyleTwitter.css

```diff
@@ -0,0 +1,630 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+
+#container{
+    display: grid;
+    grid-template-columns: 1fr  2fr auto;
+    grid-template-rows: 100vh;
+    grid-template-areas: 
+    "left main right"
+}
+
+#left-bar{
+    display: grid;
+    grid-area: left;
+    border: 2px solid rgb(228, 228, 228);
+    padding-left: 50px;
+}
+
+#twitter-logo > svg{
+    width: 30px;
+   
+    margin: 10px;
+    margin-left: 20px;
+    fill: rgb(29, 155, 240);
+}
+
+#home{
+    display: grid;
+    grid-template-columns: 30px auto;
+    margin: 10px;
+    margin-left: 20px;   
+    text-decoration: none;
+    color: black;
+    width: fit-content
+}
+
+#home:hover{
+    fill: rgb(29, 155, 240);
+    color: rgb(29, 155, 240);
+    cursor: pointer;
+}
+
+#home-text{
+    margin: 8px;
+    padding-left: 5px; 
+    
+}
+
+#explore{
+    display: grid;
+    grid-template-columns: 30px auto;
+    margin: 10px;
+    margin-left: 20px;
+    text-decoration: none;
+    color: black;
+    width: fit-content
+
+}
+
+#explore:hover{
+    fill: rgb(29, 155, 240);
+    color: rgb(29, 155, 240);
+    cursor: pointer;
+}
+
+
+#explore-text{
+   
+    margin: 8px;
+    padding-left: 5px;
+
+    
+}
+
+#notifications{
+    display: grid;
+    margin: 10px;
+    margin-left: 20px;
+    grid-template-columns: 30px auto;
+    text-decoration: none;
+    color: black;
+    width: fit-content
+
+}
+
+#notifications:hover{
+    fill: rgb(29, 155, 240);
+    color: rgb(29, 155, 240);
+    cursor: pointer;
+}
+
+#notifications-text{
+    
+    margin: 8px;
+    padding-left: 5px;
+
+}
+
+
+#messages{
+    display: grid;
+    margin: 10px;
+    margin-left: 20px;
+    grid-template-columns: 30px auto;
+    text-decoration: none;
+    color: black;
+    width: fit-content
+
+}
+
+#messages:hover{
+    fill: rgb(29, 155, 240);
+    color: rgb(29, 155, 240);
+    cursor: pointer;
+}
+
+#messages-text{
+    
+    margin: 8px;
+    padding-left: 5px;
+
+    
+}
+
+
+#bookmarks{
+    display: grid;
+    margin: 10px;
+    margin-left: 20px;
+    grid-template-columns: 30px auto;
+    text-decoration: none;
+    color: black;
+    width: fit-content
+
+}
+
+#bookmarks:hover{
+    fill: rgb(29, 155, 240);
+    color: rgb(29, 155, 240);
+    cursor: pointer;
+}
+
+#bookmarks-text{
+    
+    margin: 8px;
+    padding-left: 5px;
+
+    
+}
+
+
+#lists{
+    display: grid;
+    margin: 10px;
+    margin-left: 20px;
+    grid-template-columns: 30px auto;
+    text-decoration: none;
+    color: black;
+    width: fit-content
+
+}
+
+#lists:hover{
+    fill: rgb(29, 155, 240);
```

Comment:
Never used the fill property before, how does it differ from the background color / color property?